src/views/partials/breaches.js

```diff
@@ -99,7 +105,17 @@ export const breaches = data => `
 </section>
 <section class='breaches-table' data-token=${data.csrfToken}>
   <header>
-    <span>${getMessage('column-company')}</span><span>${getMessage('column-breached-data')}</span><span>${getMessage('column-detected')}</span>
+    <span>${getMessage('column-company')}</span>
+    <span>${getMessage('column-breached-data')}</span>
+    ${
+      /*
+       * The active/resolved badge does not have a column header, but by
+       * including an empty <span>, we can re-use the `nth-child`-based
+       * selectors for the content columns.
+       */
+      '<span></span>'
```

Comment:
Sort of a bummer, but since there's no user impact nor real maintainability impact, I don't think it's worth the effort.

The bigger bummer is that the current markup isn't really accessible. If there had been an actual relation between the headers and the columns, for example, I'd have added a screen-reader-only status label here. But since there isn't, just going with what works seems fine to me.

Reply:
Sounds good.  Yes, we have an accessibility audit ticket in the queue 👍